.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+Les ***issues*** sont rédigées en markdown. GitHub à écrit un bon résumer de comment se servir du
+markdown [ici](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).
+
+## Prérequis
+
+***Cette partie est juste là pour vous rappeler les prérequis avant de créer une pull request, vous pouvez la supprimer
+avant sa création***
+
+Merci pour votre contribution :heart:. Avant de créer la **pull request** merci d'avoir vérifié que vous remplissez
+toutes les conditions de la partie ["Liste de vérification"](#Checks). De surcroît, merci de vérifier que vous avez bien
+remplis toutes les parties qui suivent et que vous respectez les règles de titre ceci rend le travail des **reviewer**
+plus simples :heart:.
+
+### Titre des "pull request" :
+
+Votre titre de **pull request** doit contenir un libellé ainsi qu'une description brève. Les libellés doivent être
+pertinents et minimaliste, voici une liste non exhaustive :
+
+- [BOGUE] : Bogue réglé.
+- [FONCTIONNALITÉ] : Ajout d'une nouvelle fonctionnalité.
+- [CASSE] : Ajouté ce libellé pour toute PR qui casserai la rétro compatibilité.
+- [DOCUMENTATION] : mise à jour de la documentation.
+
+Exemple de titre : `[BOGUE] Les morceaux d'annanas sur la pizza ont été enlevés`
```

Comment:
Il me semble qu'il est possible de mettre plusieurs template de PR.
Je pense que ça peut être plus intéressant que de mettre un pseudo tutoriel dans un seul template.

Créer un template par type de PR, avec les titres par conséquent préremplis.
Cela permettrait aussi de mettre des checks différents ou des champs à remplir en plus.

Reply:
Oui, c'est possible, mais leur utilisation est très contraignante. Quand on n'a qu'un seul modèle de PR, il est automatiquement appliqué. Alors que quand tu en fait plusieurs, il faut utiliser des query dans l'url c'est très chiant.

[Cf Stackoverflow la deuxième réponse](https://stackoverflow.com/questions/73771068/multiple-templates-for-pull-requests-on-github)

Reply:
Ok je pensais que c'était comme dans GitLab avec une dropdown.
Dans ce cas c'est tout bon.